website/default.twig

```diff
@@ -7,12 +7,12 @@
     <link rel="stylesheet" href="//maxcdn.bootstrapcdn.com/font-awesome/4.3.0/css/font-awesome.min.css">
     <link href='https://fonts.googleapis.com/css?family=Lato:400,700,400italic,700italic|Roboto+Slab:400,700|Inconsolata:400,700&subset=latin,cyrillic' rel='stylesheet' type='text/css'>
 
-    <link rel="stylesheet" href="{{ baseUrl }}/css/theme.css" type="text/css"/>
-    <link rel="stylesheet" href="{{ baseUrl }}/css/theme-fixes.css" type="text/css"/>
+    <link rel="stylesheet" href="/css/theme.css" type="text/css"/>
+    <link rel="stylesheet" href="/css/theme-fixes.css" type="text/css"/>
     <link rel="stylesheet" href="//cdnjs.cloudflare.com/ajax/libs/highlight.js/8.4/styles/github.min.css">
 
     <link href="//fonts.googleapis.com/css?family=Open+Sans:300,400" rel="stylesheet">
```

Comment:
we're not supporting https at this time. i'll make a new ticket to track that separately. also these links need to be relative instead of using `baseUrl` which is the absolute path to pelias.io.